### ISSUE TITLE
Can't figure out how to play Super Famicom CX4 games in Higan core

### ISSUE THREAD

**Issue by MicheleKusanagi** [OpenEmu/OpenEmu#1247]
Hi, I'm new here ^^ Thank you for this wonderful project that gives emulation a whole new dignity on Macs!

I'm seriously planning to move my whole emulation activity on OpenEmu, but there's an issue I don't know how to fix: making the Higan core able to run Super Famicom CX4 games -Rockman X2 and Rockman X3.

I have a separate CX4.rom file, and I tried to place it everywhere, from drag 'n' dropping it into the library -it gets recognized as an arcade rom file O_O- to navigating to the OpenEmu Application Support folder, where I found a Higan/Super Famicom folder that includes games folders for the two games I've mentioned, I replaced the zero-byte cx4.data.rom I found there with the 3072 bytes cx4.rom I have and renamed it cx4.data.rom, only to get the same infamous "helper application couldn't communicate" error message and see that file turned into a zero-byte file.

I know SNES9x runs Rockman X2 and X3 well, but still, I'd love to stick to the Higan core for my Super Famicom emulation needs. Could you please help me?

I'm using OpenEmu 1.0.2 experimental, and my my Mac is an early 2011 Macbook Pro running Mavericks 10.9.2

Thank you in advance for your support ^^

**Comment by ddbehsnilian** [comment]
Hi, I'm having the same problem as MicheleKusanagi, but copying the cx4.rom file into the ~/Library/Application Support/OpenEmu/BIOS folder did not work for me this time. I find it strange since I did the same thing with the PlayStation BIOS roms and they worked fine. I know my chip dumps are correct because I checked the md5 checksums against the ones in the BIOS section of the User Guide (https://github.com/OpenEmu/OpenEmu/wiki/User-guide:-BIOS-files#snes-chip-dumps-for-higan). Also I have checked and Higan is the selected core for MegaMan X2 and X3.
I have restarted both OpenEmu and my computer but I am still unable to play MegaMan X2 or X3 (black screen). Using Experimental version 1.0.4 554-gf588330-Release
